docs/source/addons/theme.md

```diff
@@ -0,0 +1,220 @@
+
+```json
+"theme": "volto-my-theme"
+```
+
+or via a `THEME` variable:
+
+```shell
+THEME='volto-my-theme' yarn start
+```
+
+2. Create a directory `src/theme` in your add-on, then add this file `theme.config`, replacing `<name_of_your_theme>` with your add-on name:
+
+```less
+/*******************************
+        Theme Selection
+*******************************/
+
+/* To override a theme for an individual element specify theme name below */
+
+/* Global */
+@site        : 'pastanaga';
+@reset       : 'pastanaga';
+
+/* Elements */
+@button      : 'pastanaga';
+@container   : 'pastanaga';
+@divider     : 'pastanaga';
+@flag        : 'pastanaga';
+@header      : 'pastanaga';
+@icon        : 'pastanaga';
+@image       : 'pastanaga';
+@input       : 'pastanaga';
+@label       : 'pastanaga';
+@list        : 'pastanaga';
+@loader      : 'pastanaga';
+@placeholder : 'pastanaga';
+@rail        : 'pastanaga';
+@reveal      : 'pastanaga';
+@segment     : 'pastanaga';
+@step        : 'pastanaga';
+
+/* Collections */
+@breadcrumb  : 'pastanaga';
+@form        : 'pastanaga';
+@grid        : 'pastanaga';
+@menu        : 'pastanaga';
+@message     : 'pastanaga';
+@table       : 'pastanaga';
+
+/* Modules */
+@accordion   : 'pastanaga';
+@checkbox    : 'pastanaga';
+@dimmer      : 'pastanaga';
+@dropdown    : 'pastanaga';
+@embed       : 'pastanaga';
+@modal       : 'pastanaga';
+@nag         : 'pastanaga';
+@popup       : 'pastanaga';
+@progress    : 'pastanaga';
+@rating      : 'pastanaga';
+@search      : 'pastanaga';
+@shape       : 'pastanaga';
+@sidebar     : 'pastanaga';
+@sticky      : 'pastanaga';
+@tab         : 'pastanaga';
+@transition  : 'pastanaga';
+
+/* Views */
+@ad          : 'pastanaga';
+@card        : 'pastanaga';
+@comment     : 'pastanaga';
+@feed        : 'pastanaga';
+@item        : 'pastanaga';
+@statistic   : 'pastanaga';
+
+/* Extras */
+@main        : 'pastanaga';
+@custom      : 'pastanaga';
+
+/*******************************
+            Folders
+*******************************/
+
+/* Path to theme packages */
+@themesFolder : '~volto-themes';
+
+/* Path to site override folder */
+@siteFolder  : "~<name_of_your_theme>/theme";
+
+/*******************************
+         Import Theme
+*******************************/
+
+@import (multiple) "~semantic-ui-less/theme.less";
+@fontPath : "~volto-themes/@{theme}/assets/fonts";
+
+.loadAddonOverrides() {
+  @import (optional) "@{siteFolder}/@{addon}/@{addontype}s/@{addonelement}.overrides";
+}
+
+/* End Config */
+```
+
+3. Declare the theme as an add-on by adding its name as the value for the `addons` key in either `volto.config.js` or `package.json`.
+4. After starting Volto, the theme should be active.
+   Now you can add overrides to the default theme in `src/theme`, same as you would in a project.
+5. Now you can safely delete your project's `theme` folder, since the one in the add-on will take precedence.
+
+## Using your own escape hatch
```

Comment:
Maybe start with a first line about why and when a hatch is needed. From an integrater point of view, without having built multiple themes before.